src/webview_provider/queryResultPanel.ts

```diff
@@ -336,6 +336,10 @@ export class QueryResultPanel implements WebviewViewProvider {
       const output = await queryExecution.executeQuery();
       await this.transmitDataWrapper(output, query);
     } catch (exc: any) {
+      const activeFile = window.activeTextEditor?.document.uri;
+      if (activeFile) {
+        this.dbtProjectContainer.showCompiledSQL(activeFile);
```

Comment:
This method has nothing to with executed sql, it will also only work if the user has run `dbt compile/build/run -m <model>` before. it will refer to the compiled sql of that time.